pkgs/applications/misc/wordbook/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, fetchFromGitHub
+, python3
+, meson
+, ninja
+, pkg-config
+, glib
+, gtk4
+, libadwaita
+, librsvg
+, espeak-ng
+, gobject-introspection
+, wrapGAppsHook4
+, appstream-glib
+, desktop-file-utils
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "wordbook";
+  version = "unstable-2022-08-30";
+
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "fushinari";
+    repo = "Wordbook";
+    rev = "f72d9e748b7e182649dc18cefe27f93cf674e4b3";
+    hash = "sha256-ZL2ZgTezgH6wXxEgI2YAYOjX/DPb5irAAvyz7XGFl+U=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook4
+    appstream-glib
+    desktop-file-utils
+  ];
+
+  buildInputs = [
+    glib
+    gtk4
+    librsvg
+    libadwaita
+    gobject-introspection
```

Comment:
You are sure this is not only required in nativeBuildInputs?

Reply:
It still need `gobject-introspection` in buildInputs, when build this with newest commit.

Reply:
Please add a very short comment why so that it is not accidentally removed

Reply:
weird, but build passed this time.